apps/contracts/contracts/Constants.sol

```diff
@@ -11,9 +12,9 @@ contract Constants {
 
     // This array represents the end of each state
     // For example: Verification ends at the start of 42nd day
-    // Thus stateLength are [42, 15, 2, 1]
-    // Thereby making the Governance Cycle 42+15+2+1=60 days long
-    uint8 [] STATE_COMPLETION_LENGTHS = [42, 57, 59, 60];
+    // Thus stateLength are [1, 41, 15, 2, 1]
+    // Thereby making the Governance Cycle 1+41+15+2+1=60 days long
+    uint8 [] STATE_COMPLETION_LENGTHS = [1, 42, 57, 59, 60];
```

Comment:
The values in this line are seconds or days?

Reply:
The values are in days